src/Mgate/PersonneBundle/Resources/translations/person.fr.yml

```diff
@@ -0,0 +1,155 @@
+generic:
```

Comment:
It does.

One day maybe, Jeyser will be an flourishing, multi country project. For the moment it is managed by 2 french juniors-entreprises, by frenchs developers for french users. 

If keys are in english, meaning is less obvious for the totality of developers involved on the project. 
It is not necessary to add additional difficulty to the project.
So ok it's not as standard as it could be, but isn't it better (as for french) to cope with `personne.filiere_modifier` rather than `people.pathways_edit` ?